{"name": "Проверка негативного сценариея принятия заказа с неправильный ID", "status": "passed", "description": "Фикстура создает курьера и заказ, после проеряется негативный сценарий принятия заказа {negative_data}", "attachments": [{"name": "stdout", "source": "84221a47-5d37-48ab-93a5-86e87955e338-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "negative_data", "value": "{'courier_id': True, 'order_id': 99999999, 'code': 404, 'message': 'Заказа с таким id не существует'}"}], "start": 1757920942479, "stop": 1757920942895, "uuid": "c61721c1-7129-42a2-a6f2-2fa3b1c59915", "historyId": "32e748799898ce0ac404fe842e34eae9", "testCaseId": "3eb2b6a91772a0b54f59ffa7d7a7b071", "fullName": "tests.test_accept_order.TestAcceptOrder#test_wrong_ids_negative_check_response", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "RUMS01CW-F915A8"}, {"name": "thread", "value": "12672-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}], "titlePath": ["tests", "test_accept_order.py", "TestAcceptOrder"]}